pkg/api/controller.go

```diff
@@ -82,6 +82,14 @@ type Controller struct {
 	Emailer               *email.Emailer
 }
 
+func (c *Controller) GetAuthCapabilities(w http.ResponseWriter, r *http.Request) {
+	emailSupported := c.Emailer.Params.SMTPHost != ""
```

Comment:
Nit: Is there a better way to switch off email than to remove some field from the configuration?  Many features allow explicitly disabling or enabling them.  That would allow me "just" to disable email without having to delete or comment-out the SMTP host configuration (which could be tricky to remember)

Reply:
The purpose here is the opposite, we want to verify the user definitely set up an email server and the most simple way is to see if one of such was configured. I do agree we should have such a switch to enable and disable but I don't think this is in the scope of this PR.